### ISSUE TITLE
Reload Image Dataset XML without closing/reopening Mastodon

### ISSUE THREAD

**Issue by stefanhahmann** [mastodon-sc/mastodon#246]
# Story

A user of Mastodon wants to have a refresh button in Mastodon, that would re-read the dataset xml file again without the need to close and open the active mastodon tracking project used for tracking.
The reason for this request is that the user can acquire data with OpenSPIM in the BigDataViewer format and at every new time-point the xml file is updated accordingly, which allows him/her to track nuclei while imaging an embryo.

# Acceptance Criteria

# Tasks

**Comment by maarzt** [comment]
I have an idea how to implement a simple solution. There could be a menu entry. "Save & reload", which saves the current status of the Mastodon project, closes Mastodon, and open the project again. Not sure if that would be good enough for the specific use case. But we should give it a try before implementing something a lot more complicated.

**Comment by stefanhahmann** [comment]
> I have an idea how to implement a simple solution. There could be a menu entry. "Save & reload", which saves the current status of the Mastodon project, closes Mastodon, and open the project again. Not sure if that would be good enough for the specific use case. But we should give it a try before implementing something a lot more complicated.

Good idea. From the perspective of the user, this is not exactly the same as it was requested originally. The closing / re-opening of the UI may still feel inconvenient to the user even though it would happen automatically. Not, sure how much more complicated the solution without closing / re-opening of Mastodon would be.
